src/components/enterprise-user-subsidy/enterprise-offers/data/hooks.js

```diff
@@ -48,48 +41,21 @@ export const useEnterpriseOffers = ({
     }
   }, [enterpriseId, enableOffers]);
 
-  // Fetch enterprise coupons to determine if the enterprise offers can be used to enroll
   useEffect(() => {
-    const fetchEnterpriseCoupons = async () => {
```

Comment:
note: i'm not sure why this logic is here, but it's not really being used at all.